org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php

```diff
@@ -128,3 +128,10 @@ function bootstrapcivicrm_civicrm_alterSettingsFolders(&$metaDataFolders = NULL)
 function bootstrapcivicrm_civicrm_pageRun($page) {
   CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css');
 }
+
+/**
+ * This hook is invoked after all the content of a CiviCRM form or page is generated. It allows for direct manipulation of the generated content.
+ */
+function bootstrapcivicrm_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
+  $content = '<script src="'.CRM_Extension_System::singleton()->getMapper()->keyToUrl('org.civicrm.bootstrapcivicrm').'/js/radio-checkbox.js"></script>'. $content;
```

Comment:
Ok I see why:

> CiviCRM's pages are classified as either 'Forms' or 'Pages', and this only runs on pages classified as 'Pages'.

_(from the [_pageRun wiki page](https://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_pageRun))_

So the "export contact" page is treated as a form, and for that there is the [_buildForm hook](https://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_buildForm)

You could do something like

```php
function bootstrapcivicrm_civicrm_buildForm($formName) {
  // if you want to limit the script only to specific forms..
  if ($formName == 'CRM_Contact_Form_Search_Advanced') {  
    // inject the script
  }
}
```